tests/simulate/test_lammps.py

```diff
@@ -246,6 +246,28 @@ def test_minimization(self):
         lmp.operations = emin
         lmp.run(potentials=pot, directory=self.directory)
 
+        # Whole number of iterations and evalulations as float
+        emin.max_iterations = 1000.0
```

Comment:
Did you confirm these tests fail without the bugfix?

Reply:
yes